tools/squidpy/squidpy_spatial.py

```diff
@@ -38,6 +38,15 @@ def main(inputs, anndata, output, output_plot):
     tool_func = getattr(sq.gr, tool)
 
     options = params['analyses']['options']
+    progress_bar_tools = [
```

Comment:
I think you can also try `export TQDM_DISABLE=true` to disable it globally for the tool.

Reply:
That's a much simpler solution, thanks!